NineChronicles.Headless.AccessControlCenter/Controllers/AccessControlServiceController.cs

```diff
@@ -17,22 +18,28 @@ public AccessControlServiceController(IMutableAccessControlService accessControl
         }
 
         [HttpGet("entries/{address}")]
-        public ActionResult<bool> IsAccessDenied(string address)
+        public ActionResult<int?> GetTxQuota(string address)
         {
-            return _accessControlService.IsAccessDenied(new Address(address));
+            return _accessControlService.GetTxQuota(new Address(address));
         }
 
-        [HttpPost("entries/{address}/deny")]
-        public ActionResult DenyAccess(string address)
+        [HttpPost("entries/add-tx-quota/{address}/{quota:int}")]
+        public ActionResult AddTxQuota(string address, int quota)
         {
-            _accessControlService.DenyAccess(new Address(address));
+            var maxLimit = 10;
+            if (quota > maxLimit)
+            {
+                return BadRequest($"The limit cannot exceed {maxLimit}.");
```

Comment:
It isn't critical but I think it is good to unify terms. How about using `quota` instead of `limit`?

Reply:
@moreal You mean `maxLimit` to `maxQuota` right?

Reply:
Ah, it is a good point too. I mean the message in the `BadRequest` function's argument because it is shown to users.
```suggestion
                return BadRequest($"The quota cannot exceed {maxLimit}.");
```

Reply:
Ah i see it now. I'll unify the terms, thank you!